client/src/components/util/truncate.js

```diff
@@ -86,18 +86,12 @@ export default (props) => {
     })
   );
   return (
-    <Tooltip
+    <Tooltip2
```

Comment:
I noticed that the old user annotation code is using this module, and it uses the old Popover component.  Is popover compatible with Tooltip2?

Put another way, have we confirmed that this change does not break category user annotations?

Reply:
![image](https://user-images.githubusercontent.com/1770265/112069355-1288f500-8b42-11eb-89e3-27734a379c4e.png)